brew_view/controllers/system_list_api.py

```diff
@@ -42,20 +63,48 @@ def get(self):
         tags:
           - Systems
         """
-        self.logger.debug("Getting all Systems")
-
+        query_set = System.objects.order_by(self.request.headers.get('order_by', 'name'))
+        serialize_params = {'to_string': True, 'many': True}
+
+        include_fields = self.request.headers.get('include_fields')
+        exclude_fields = self.request.headers.get('exclude_fields')
+        include_commands = self.get_query_argument('include_commands', None)
+
+        if include_fields and exclude_fields:
```

Comment:
This seems a little arbitrary. Let's just pick one and document it.

I pick:

included_fields are guaranteed to be included, excluded fields will be excluded unless explicity asked for.

Reply:
Alrighty

Reply:
I'm reversing this because that's what mongoengine does. So excluded fields are guaranteed to not be there, included fields will be there unless specifically excluded.